keyboards/matrix/noah/matrix.c

```diff
@@ -151,16 +151,4 @@ matrix_row_t matrix_get_row(uint8_t row) { return matrix[row]; }
 
 void matrix_print(void)
 {
```

Comment:
```suggestion
{
    #ifdef CONSOLE_ENABLE
```
Instead of removing, only compile if `printf` is available.